core/modules/date/js/date-range.js

```diff
@@ -140,10 +140,10 @@ Backdrop.date.EndDateHandler.prototype.startClickHandler = function () {
 Backdrop.date.EndDateHandler.prototype.endFocusHandler = function (event) {
   for (var id in this.selects) {
     if (this.selects.hasOwnProperty(id)) {
-      this.selects[id].start.unbind('click.endDateHandler');
+      this.selects[id].start.off('click.endDateHandler');
     }
   }
-  $(event.target).unbind('focus', this.endFocusHandler);
+  $(event.target).off('focus');
```

Comment:
I noticed here that you removed the handler name when you converted this to `off`. I believe the handler name is needed here. That way you unbind only that event handler from the event. As it is right now, this will remove all listeners to the `focus` event.

Reply:
Good reminder: this is something where I want to take a second and closer look. :+1:

Reply:
Added it back :+1: